waku/v2/node/config.nim

```diff
@@ -47,6 +52,10 @@ type
       defaultValue: 50
       name: "max-connections" }: uint16
     
+    transport* {.
+      desc: "Specify transport to use for wakunode " &
```

Comment:
This should allow for multiple transports. We don't want to add support for Websockets and remove support for TCP.